### ISSUE TITLE
Autoptimize JavaScript Optimization Removing Header and Stripping Tags from Page

### ISSUE THREAD

**Issue by taylorkspencer** [futtta/autoptimize#166]
There is a bug in the "Optimize JavaScript" function of Autoptimize that is removing the header and stripping the closing `</head>` and opening `<body>` tags from select pages. This can be seen on https://www.collaborativedivorceneworleans.com/faqs/, where the header that is usually visible on this site is not seen on this page, and when you go into the View Source, the closing `</head>` and opening `<body>` tags cannot be found.  I initially noticed this bug in the stable version of Autoptimize, 2.3.4, but it is present in the latest beta, 2.4 Beta 3, as well.

**Comment by futtta** [comment]
I'll close here and we can continue [in your wordpress.org topic](https://wordpress.org/support/topic/autoptimize-javascript-optimization-removing-header-and-stripping-tags-from-page/) Taylor.

**Comment by taylorkspencer** [comment]
I found out the cause of my problem.  Apparently Autoptimize was attempting to minimize a JavaScript file that was being inserted at the theme level, `jquery.expend2.js`.  As the name implies, it's a JQuery file, which Autoptimize already has rules for excluding, but the built-in rules did not cover this one.  I added it to the exclude list and the FAQ page began displaying normally, header, closing `</head>`, opening `<body>` tags and all.  I would like to thank @futtta for the help you gave dealing with this matter.
